lib/views/video_view/custom_video_control_bar.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/base/networking/api/gocast/api_v2.pb.dart';
+
+class CustomVideoControlBar extends StatelessWidget {
+  final Function(String, Stream) onMenuSelection;
+  final VoidCallback onToggleChat;
+  final VoidCallback onOpenQuizzes;
+  final Stream currentStream;
+
+  const CustomVideoControlBar({
+    super.key,
+    required this.onMenuSelection,
+    required this.onToggleChat,
+    required this.onOpenQuizzes,
+    required this.currentStream,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    bool isIOS = Theme.of(context).platform == TargetPlatform.iOS;
+    return Container(
+      padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 16),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          // Icons on the left
+          Row(
+            children: [
+              IconButton(
+                icon: const Icon(Icons.messenger_outline_outlined),
+                onPressed: onToggleChat,
+              ),
+              IconButton(
+                icon: const Icon(Icons.quiz_outlined),
+                onPressed: onOpenQuizzes,
+              ),
+            ],
+          ),
+
+          // Three-dot menu on the right
+          PopupMenuButton<String>(
+            onSelected: (choice) => onMenuSelection(choice, currentStream),
+            itemBuilder: (BuildContext context) {
+              return <Map<String, IconData>>[
```

Comment:
We need also the 'Split view' icon which is shown whenever both presentation and camera view are available (= the view where you can manually resize the presentation and camera view next to each other)

Reply:
Done ✅